README.md

```diff
@@ -1,5 +1,7 @@
 # FlowForge
 
+https://flowforge.com/
```

Comment:
This floating at the top of the docs on GitHub looks out of place.

GitHub has an inbuilt repository about section, that's likely a better place for this:

![Screenshot 2023-06-27 at 15 58 00](https://github.com/flowforge/flowforge/assets/507155/a92a8051-672e-4c2c-9871-081cb0c828b0)

Someone with the correct GitHub permissions can add the link.

Reply:
There are a few places we can add the link... but having it in the readme is important (aside from exact placement etc) as that will also end up on the npm page for our package - https://www.npmjs.com/package/@flowforge/flowforge

Reply:
On NPM we could also update the website field to actually point to the website. I believe that's pulled from package.json's homepage field.

Reply:
I've updated the github website field (and added `node-red` and `low-code` keywords)

Reply:
I have updated the `homepage` field and added the company logo as the link to the homepage in the readme.